praw/reddit.py

```diff
@@ -485,7 +485,7 @@ def info(
                   different set of submissions.
 
         """
-        none_count = [fullnames, url].count(None)
+        none_count = (fullnames, url).count(None)
```

Comment:
Why was this changed? We should standardize these to either use lists or tuples, and right now, most of the models use lists.

Reply:
No reason really. I just feel like I have the freedom to since I put this line in and at the time I didn’t realise tuples had `.count()`. It would have made things a tiny bit more efficient for Python 2 users.

Reply:
You're right, it's significantly faster by a magnitude of 33%.

Reply:
Over 1000000 trials.

I thought I read somewhere this got optimised away in Python 3 but maybe not. Looks like tuple `.count()` is the way to go.